tools/cli/commands.js

```diff
@@ -1192,10 +1192,8 @@ main.registerCommand({
   }
   
   if (process.env.MONGO_URL) {
-    Console.error("meteor reset won't reset the DB provided by MONGO_URL, as it can be dangerous " +
-                  "if you specify MONGO_URL to your production server. If you are sure to do that, " +
-                  "do it yourself by raw mongo queries.");
-    return 1;
+    Console.info("meteor reset won't reset the DB provided by MONGO_URL, while other project" +
+                 "things get reset. If you are sure to reset DB, do it yourself by raw mongo queries.");
```

Comment:
And I would personally recommend this for the text, but maybe wait for more input before you change the PR:
```
As a precaution, `reset` only clears the local, Meteor-provided database.  The database specified with `MONGO_URL` will *NOT* be reset and would need to be reset manually.
```